portality/templates/email/publisher_application_inprogress.txt

```diff
@@ -1,6 +1,6 @@
 Dear {{ publisher_name }},
 
-The application or update request for "{{ journal_title }}" is now being reviewed by an Associate Editor.
+The update request for "{{ journal_title }}" is now being reviewed by an Associate Editor.
```

Comment:
This PR doesn't seem to have the `publisher_update_request_*.txt` templates included

Reply:
That's really strange. I definitely added them here! I will fix it in a moment.